src/merkletree.rs

```diff
@@ -127,6 +127,10 @@ impl MerkleTree {
     pub fn iter(&self) -> std::collections::hash_map::Iter<Hash, Value> {
         self.kvs.iter()
     }
+
+    pub fn kvs(&self) -> &HashMap<Hash, Value> {
```

Comment:
To verify the Pod ID of a MockSignedPod we need to recalculate the root.  To recalculate the root we need to create a new MerkleTree object.  The MerkleTree constructor takes a HashMap, so I found it easier to add this method to easily recalculate the root of an existing MerkleTree object like this:
```rust
        let new_mt = MerkleTree::new(mt.kvs());
        let new_id = new_mt.root();
```

In the future we can have a generic MerkleTree constructor that takes an iterator, and then we can do
```rust
        let new_mt = MerkleTree::new(mt.iter());
        let new_id = new_mt.root();
```